crates/dbsp/src/operator/dynamic/controlled_filter.rs

```diff
@@ -297,8 +298,10 @@ mod test {
                     |x, y| *max(x, y),
                 );
 
-                let (output, errors) = input.controlled_key_filter(
-                    &threshold,
+                let (output, errors) = input.controlled_key_filter_typed(
+                    &threshold.apply(|b: &TypedBox<Tup1<u64>, DynData>| -> Tup1<u64> {
```

Comment:
should this also be a method that is the inverse of typed_box?

Reply:
There is one already. Whatever was I thinking?